jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpStatus.java

```diff
@@ -25,7 +25,9 @@ public class HttpStatus
     public static final int CONTINUE_100 = 100;
     public static final int SWITCHING_PROTOCOLS_101 = 101;
     public static final int PROCESSING_102 = 102;
+    @Deprecated(forRemoval = true)
```

Comment:
Can we also have `since` ...

```suggestion
    @Deprecated(forRemoval = true, since = "12.0.1")
```